net/netcheck/netcheck.go

```diff
@@ -675,11 +729,13 @@ func (rs *reportState) addNodeLatency(node *tailcfg.DERPNode, ipp netip.AddrPort
 		updateLatency(ret.RegionV6Latency, node.RegionID, d)
 		ret.IPv6 = true
 		ret.GlobalV6 = ipPortStr
+		mak.Set(&ret.GlobalV6Counters, ipPortStr, ret.GlobalV6Counters[ipPortStr]+1)
```

Comment:
We have an `AddrPort` in `ipp`, if you wanted to use that as the map key...

Reply:
Turns out that this gets a bit messy and I removed all the stringly addrports as a result, net win.

Reply:
Oh awesome 🎉